lib/lambda_deployment/configuration.rb

```diff
@@ -12,6 +14,9 @@ def load_config(config_file)
       @s3_bucket = config.fetch('s3_bucket', ENV.fetch('LAMBDA_S3_BUCKET', nil))
       @s3_key = s3_key_name(config.fetch('file_name'))
       @s3_sse = config.fetch('s3_sse', ENV.fetch('LAMBDA_S3_SSE', nil))
+
+      @config_env = config.fetch('environment', {})
+      @kms_key_arn = config.fetch('kms_key_arn', ENV.fetch('LAMBDA_KMS_KEY_ARN', nil))
```

Comment:
FYI same as `config['kms_key_arn'] || ENV['LAMBDA_KMS_KEY_ARN']` ... with the added benefit that it short-circuits